pkg/operator/ceph/csi/version.go

```diff
@@ -33,7 +33,16 @@ var (
 	releasev310          = CephCSIVersion{3, 1, 0}
 	releasev320          = CephCSIVersion{3, 2, 0}
 	releasev330          = CephCSIVersion{3, 3, 0}
-	supportedCSIVersions = []CephCSIVersion{minimum, releaseV210, releasev300, releasev310, releasev320, releasev330}
+	releasev340          = CephCSIVersion{3, 4, 0}
+	supportedCSIVersions = []CephCSIVersion{
+		minimum,
+		releaseV210,
+		releasev300,
```

Comment:
Is this a good time to bump the min version to 3.0? I doubt very many people are changing the default version installed with rook, though I don't know for sure. This seems like a reasonable change for 1.7. Thoughts?

Reply:
Yes, there is no major change but good to update to 3.x and also it will also make sure people are using the latest major release.